.github/workflows/docker.yml

```diff
@@ -108,6 +111,7 @@ jobs:
           build-args: |
             VERSION_TAG=${{ steps.prep.outputs.version }}
           load: true
+          platforms: linux/amd64,linux/arm64
```

Comment:
To avoid getting 
```
ERROR: docker exporter does not currently support exporting manifest lists
```
you'll have to either delete this line, or create a third copy of the `docker/build-push-action@v3`, (one with load, one with platforms, one with push).